docs/src/configuring-languages.md

```diff
@@ -57,7 +57,14 @@ You can customize a wide range of settings for each language, including:
 - `preferred_line_length`: The recommended maximum line length
 - `soft_wrap`: How to wrap long lines of code
 
-These settings allow you to maintain specific coding styles across different languages and projects.
+These settings allow you to maintain specific coding styles across different languages and projects. Specifically, the field `soft_wrap` can be:
+
+- `none`: Do not soft wrap.
```

Comment:
We already have the values of `soft_wrap` documented here: https://github.com/zed-industries/zed/blob/cf0a8a7a1a47cfda5a48348d8309a00a737f2b1e/docs/src/configuring-zed.md#L1182-L1193

I don't think it makes sense to duplicate them here as well.